src/services/event-service.ts

```diff
@@ -149,13 +154,14 @@ export class EventService<E = any> {
 
 
     const reduceEvents = (previous, {address, ...rest}) => {
+      address = address.toLowerCase();
       if (!previous[address])
-        previous[address] = {info: networkMap[address], returnValues: []};
+        previous[address] = {info: {...networkMap[address], chainId}, returnValues: []};
```

Comment:
I would think that `chainId` already exists inside `info` since you made all the needed relations and `networkMap[address]` is the raw result of a `db.model.networks.findAll()`